### PR TITLE
Index.md full translation to the Persian

### PR DESCRIPTION
**Pull request by mheidari988**
Created "fa" folder inside "docs" folder as per convention.
Created a copy of "images" folder from "en" to the "fa".
Translated and added "Index.md" Farsi aka Persian language version.

### CODE REVIEW

**Comment by hikalkan** [comment]
Thanks a lot for your contribution. As stated in the contribution document;

>There are some fundamental documents need to be translated before publishing a language on the [ABP documentation web site](https://docs.abp.io/):

* Index (Home)
* Getting Started
* Web Application Development Tutorial

I see the last item is missing in this translation. We can accept the new language if you can complete this translation. Thanks a lot. See the contribution guide: https://docs.abp.io/en/abp/latest/Contribution/Index#document-translation

**Comment by mheidari988** [comment]
Sure, I'm working on it. RTL languages have pain with technical documentation, But I think I can finish it soon.

**Comment by RezaPouya** [comment]
Thanks a lot . 
Do you need any help ?  
We are using abp in our project for over a year now , and we translated most of the resources for modules for ourselves .
so I have a question , are you working on translating the resources !? if yes , do you need any help !? if not , I can work on it . @hikalkan  @mheidari988

File: docs/fa/Getting-Started-AspNetCore-Application.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+app.InitializeApplication();
+
+app.Run();
+
+````
+
+## کد منبع
+
+کد منبع پروژه نمونه ایجاد شده در این آموزش را از [here](https://github.com/abpframework/abp-samples/tree/master/BasicAspNetCoreApplication).
```
Comment (ashkandehnavi) [inline docs/fa/Getting-Started-AspNetCore-Application.md]: better using "اینجا" for "here"

File: docs/fa/Getting-Started-Create-Solution.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+abp new Acme.BookStore{{if UI == "NG"}} -u angular{{else if UI == "Blazor"}} -u blazor{{else if UI == "BlazorServer"}} -u blazor-server{{end}}{{if DB == "Mongo"}} -d mongodb{{end}}{{if Tiered == "Yes"}}{{if UI == "MVC" || UI == "BlazorServer"}} --tiered{{else}} --separate-identity-server{{end}}{{end}}
+````
+
+*میتوانید از نام گذاری های دلخواه خود استفاده نمایید مثال BookStore, Acme.BookStore و یا Acme.Retail.BookStore.* 
+
+{{ if Tiered == "Yes" }}
+
+{{ if UI == "MVC" || UI == "BlazorServer" }}
+
+* `--tiered` با استفاده از این گذینه میتوانید از مدل چند لایه استفاده نمایید که لایه Authentıcatıon، لایه API و واست کاربری مجزا باشند.
```
Comment (ashkandehnavi) [inline docs/fa/Getting-Started-Create-Solution.md:32]: `--tiered` با استفاده از این گزینه میتوانید از مدل چند لایه استفاده نمایید که لایه Authentıcatıon، لایه API و رابط کاربری به صورت فیزیکی مجزا باشند.

File: docs/fa/Getting-Started-Create-Solution.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+{{ end }}
+
+{{ end }}
+
+> [ABP CLI document](./CLI.md) تمام اطلاعات مربوط به دستور ها در این لینک موجود میباشد.
+
+## توسعه نرم افزار موبایل
+
+جهت استفاده از ساختار های [React Native](https://reactnative.dev/) در پروژه خود از دستور  `-m react-native` (و یا `--mobile react-native`) استفاده نمایید. این دستور اجازه میدهد تمپلیت اولیه مربوط به ری اکت نیتیو مناسب تولید نرم افزار های تحت موبایل به پروژه اضافه شود
```
Comment (ashkandehnavi) [inline docs/fa/Getting-Started-Create-Solution.md]: better using "span" and ltr direction for commands:

جهت استفاده از ساختار های [React Native](https://reactnative.dev/) در پروژه خود از دستور  <span dir="ltr">`-m react-native` </span> (و یا <span dir="ltr">`--mobile react-native`</span>) استفاده نمایید. این دستور اجازه میدهد تمپلیت اولیه مربوط به ری اکت نیتیو مناسب تولید نرم افزار های تحت موبایل به پروژه اضافه شود

**Review by ashkandehnavi — CHANGES_REQUESTED** [review]
hi,
please use `<span dir="ltr">left to right text</span>` example: `<span dir="ltr">.IdentityServer</span>` or `<span dir="ltr">-dbms MySQL</span>` if do not using, in rtl view clutter of texts.

**Comment by mheidari988** [comment]
> Thanks a lot . Do you need any help ? We are using abp in our project for over a year now , and we translated most of the resources for modules for ourselves . so I have a question , are you working on translating the resources !? if yes , do you need any help !? if not , I can work on it . @hikalkan @mheidari988

Great,
Reference to [Contribution Guide](https://docs.abp.io/en/abp/latest/Contribution/Index#contribution-guide) and @hikalkan's [comment ](https://github.com/abpframework/abp/pull/12160#issuecomment-1097237212) there is two way to contribute which I believe if we can translate some documents, may be a great way to help Iranian developers to leverage the power of ABP Framework.

- [Resource Localization](https://docs.abp.io/en/abp/latest/Contribution/Index#resource-localization) is essential if anyone tries to use full feature ABP in Farsi language aka `Persian`, and I am currently working on it and I can share `abp-translate.json` so we split the task and make it done soon. By the way any help is appreciated.
-  Second is about [Document Translation](https://docs.abp.io/en/abp/latest/Contribution/Index#document-translation), which we need to translate `Web Application Development Tutorial` too.

By the way, I believe it is much better if we discuss these on ABP's official discord community. (my discord mohi#4787)

File: docs/fa/Getting-Started-AspNetCore-Application.md (hunk 1 above)
Comment (mheidari988) [reply docs/fa/Getting-Started-AspNetCore-Application.md]: done

File: docs/fa/Getting-Started-Create-Solution.md (hunk 3 above)
Comment (mheidari988) [reply docs/fa/Getting-Started-Create-Solution.md]: great point

**Comment by RezaPouya** [comment]
> 

hi , thanks for your response . Im willing to help in translating abp-translate.json. if you are using private repository , just invite me , if you are using public one , then tell me which one it is ...

**Comment by mheidari988** [comment]
> > 
> 
> hi , thanks for your response . Im willing to help in translating abp-translate.json. if you are using private repository , just invite me , if you are using public one , then tell me which one it is ...

It's much better if we translate the whole `abp-translate.json` file and then PR. so I started a temporary repo. please check [abp-farsi](https://github.com/mheidari988/abp-farsi) and I invited you and @ashkandehnavi

**Comment by ebicoglu** [comment]
good work guys! waiting the Persian docs.
thanks

**Comment by hitaspdotnet** [comment]
Good work guys! I am happy to see more Persian joining the framework community soon. Thanks.

**Comment by hikalkan** [comment]
Hi,
I want to inform you that we've decided to remove all translations of the new documentation system (https://abp.io/docs/latest/)

Because the documentation is frequently changing and none of the translations can catch these changes. Only the Chinese translation was very close to the original documentation, but it was getting our too much effort.

We suggest to you Google Translate or another tool for auto-translation.

We've integrated Google Translate for many languages already:

image

Current languages:

image

We will continue to improve that translation system for a smooth reading experience.

**Pull request closed without merging**